contributing_to_docs/doc_guidelines.adoc

```diff
@@ -1331,6 +1331,8 @@ fieldName: |-
   And it can be on multiple lines.
 ----
 - When truncating YAML, always precede an ellipsis with a pound so that the YAML is valid.
+- Use three hyphens (`---`) to separate YAML definitions in a single YAML file.
+- Use ellipsis (`...`) to mark the end of a YAML file. 
```

Comment:
I'm not sure I understand this one. Usually `...` is just used to mark that things have been truncated (as mentioned two lines above). Can you clarify what this one is supposed to mean?

Reply:
Thanks for the replies. My understanding is that the ellipses mark the end of a file without starting a new one. I'm seeing a similar explanation on many internet sources. 

Some of our docs follow this use of the ellipses, see https://docs.openshift.com/container-platform/4.12/serverless/eventing/event-sources/serverless-custom-event-sources.html#serverless-sinkbinding-reference_serverless-custom-event-sources.

YAML spec on ellipese: https://yaml.org/spec/1.2.2/#22-structures

Reply:
The 3 hyphens indicate truncated sections according to YAML rules.

Reply:
TIL about the option to use `...` to end the file. But that spec says that they are a presentation detail, so don't seem to actually be required. It also says you could use `---` to start a file, but that doesn't seem to be required either. Only if you're putting more than one in a single file.

So I don't think this is something that we need to add as a guideline then, if it's not really required, and we aren't really doing it consistently now that way. 

Also, the places where you mentioned you're seeing this in the repo, I guarantee that the majority of them are using that to mean that they have truncated other YAML fields from the example in the code block. Not that they're using it to indicate the end of the file. (Per the guidelines they should be putting a `#` in front of that, but I think a lot don't).

Reply:
Thanks for the comment. I can remove this bullet point, but perhaps I should mention with a note to not use the ellipses over the triple hyphen in the context of YAML rules?